fe/i18n/messages/pt.json

```diff
@@ -1,4 +1,26 @@
 {
+  "general": {
+    "CategoryElement": {
+      "category": {
+        "edition": { "key": "edition", "value": "Edições" },
+        "masterclass": { "key": "masterclass", "value": "Masterclasses" },
+        "workshop": { "key": "workshop", "value": "Workshops" },
+        "creative-talks": {
+          "key": "creative-talks",
```

Comment:
Why do we have key and value? Since the key is the same as the property name

Reply:
Only on English, on Portuguese it will be different 😄